### PR TITLE
Gigasecond adding hints and appending instructions

### PR DESCRIPTION
**Pull request by CodebyKhayla**
Issue #3018

### CODE REVIEW

File: exercises/practice/gigasecond/.docs/instructions.append.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Instructions append
+
+
```
Comment (BethanyG) [inline exercises/practice/gigasecond/.docs/instructions.append.md:3]:
```suggestion
## Dates and Times in Python

This exercise explores objects from Python's `datetime` module:

```

File: exercises/practice/gigasecond/.docs/instructions.append.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Instructions append
+
+  
+[Official Python documentation on the datetime module](https://docs.python.org/3.9/library/datetime.html#module-datetime)
+[datetime objects](https://docs.python.org/3.9/library/datetime.html#datetime.datetime)
+[timedelta objects](https://docs.python.org/3.9/library/datetime.html#timedelta-objects)
```
Comment (BethanyG) [inline exercises/practice/gigasecond/.docs/instructions.append.md:6]:
```suggestion
- [Official Python documentation on the datetime module][datetime]
- [datetime objects][datetime.datetime]
- [timedelta objects][datetime.timedelta]

[datetime]: https://docs.python.org/3.9/library/datetime.html#module-datetime
[datetime.datetime]: https://docs.python.org/3.9/library/datetime.html#datetime.datetime
[datetime.timedelta]: https://docs.python.org/3.9/library/datetime.html#timedelta-objects
```

File: exercises/practice/gigasecond/.docs/hints.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# Hints
+
+# General 
+* Your code should parse a datetime object, add a gigasecond's worth of time to it, and then return the result as a datetime object.
```
Comment (BethanyG) [inline exercises/practice/gigasecond/.docs/hints.md:4]:
```suggestion
-  Your code should parse a datetime object, add a gigasecond's worth of time to it, and then return the result as a datetime object. 
```

File: exercises/practice/gigasecond/.docs/hints.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# Hints
+
+# General 
+* Your code should parse a datetime object, add a gigasecond's worth of time to it, and then return the result as a datetime object. 
+
+* If you're having trouble, remember to take a look at the provided test cases under the Tests tab. These will help you figure out what the expected inputs and outputs of your function(s) should be.
```
Comment (BethanyG) [inline exercises/practice/gigasecond/.docs/hints.md:6]:
```suggestion
- If you're having trouble, remember to take a look at the provided test cases under the Tests tab. These will help you figure out what the expected inputs and outputs of your function(s) should be.
```

Comment (BethanyG) [reply exercises/practice/gigasecond/.docs/hints.md:6]: veeery small niggle here. Our internal convention is to use `-` for bullets in markdown.  Either `*` or `-` work, we just tend to use `-`.

File: exercises/practice/gigasecond/.docs/instructions.append.md (hunk 1 above)
Comment (BethanyG) [reply exercises/practice/gigasecond/.docs/instructions.append.md:3]: Thought there should be a little preamble here,  just to focus the student in on the links.

File: exercises/practice/gigasecond/.docs/instructions.append.md (hunk 2 above)
Comment (BethanyG) [reply exercises/practice/gigasecond/.docs/instructions.append.md:6]: Added bullets here to separate the links, and added the reflinks at the bottom of the doc.  If this doesn't read well, feel free to revert the bullets.  The reflinks are part of our spec, and won't show to the end user.

**Review by BethanyG — CHANGES_REQUESTED** [review]
@CodebyKhayla  - I am _delighted_ you submitting a PR!  This is excellent!  I've marked "required changes" only due to the bullet convention and the reflinks.  Everything else is a suggestion - so feel free to disagree or edit what I've put in.  As long as the bullets and reflinks are accepted, I will approve and merge.  🌟

**Comment by CodebyKhayla** [comment]
Thank you so much for all of your help @BethanyG !!! 
I have learned so much even from this experience.

**Review by BethanyG — APPROVED** [review]
(chef's kiss) Tis a thing of beauty!  @CodebyKhayla  - Thank you so much for these.  They go to production .... shortly!  🌟 🌈

**Pull request merged**